.travis.yml

```diff
@@ -1,37 +1,45 @@
-language: python
-
+os:
+- linux
+- osx
 python:
-    - "2.7"
+- '2.7'
 virtualenv:
-    # required because we install the python-qt4 ubuntu package
-    system_site_packages: true
-
+  system_site_packages: true
+before_install:
+- OS=$(if [[ "$TRAVIS_OS_NAME" = "osx" ]]; then echo "MacOSX"; else echo "Linux";
```

Comment:
I think this OS variable is causing the error, remove that line and try using TRAVIS_OS_NAME only in this file.